openedx/features/job_board/helpers.py

```diff
@@ -0,0 +1,16 @@
+from django.core.validators import ValidationError
+from django.utils.translation import ugettext as _
+
+from openedx.features.job_board.constants import LOGO_IMAGE_MAX_SIZE
+
+
+def validate_file_sie(file):
+    """
+    Validate maximum allowed file upload size, raise validation error if file size exceeds.
+    :param file: file that needs to be validated for size
+    """
+    size = getattr(file, "size", None)
+    if size and LOGO_IMAGE_MAX_SIZE < size:
```

Comment:
@nofalfayyaz it would be great if you can load `LOGO_IMAGE_MAX_SIZE` from lms.env.json

Reply:
actually the reason to put it there is because the restrictions are explicitly related to job board logo. thoughts?

Reply:
I agree with Nofal on this. We should not be creating a dependency if it is not required.